pkg/gadgets/trace/tcpdrop/tracer/bpf/tcpdrop.bpf.c

```diff
@@ -118,29 +106,20 @@ static __always_inline int __trace_tcp_drop(void *ctx, struct sock *sk, struct s
 	if (dev != NULL)
 		event.netns = BPF_CORE_READ(dev, nd_net.net, ns.inum);
 
-    struct sockets_value *skb_val = gadget_socket_lookup(sk, event.netns);
+	struct sockets_value *skb_val = gadget_socket_lookup(sk, event.netns);
 	if (skb_val != NULL) {
 		event.proc_socket.mount_ns_id = skb_val->mntns;
 		event.proc_socket.pid = skb_val->pid_tgid >> 32;
 		event.proc_socket.tid = (__u32)skb_val->pid_tgid;
 		__builtin_memcpy(&event.proc_socket.task,  skb_val->task, sizeof(event.proc_socket.task));
-    }
+	}
 
 	bpf_perf_event_output(ctx, &events, BPF_F_CURRENT_CPU, &event, sizeof(event));
 	return 0;
 }
 
-// enum skb_drop_reason is defined in the following file:
-// https://raw.githubusercontent.com/torvalds/linux/v6.2/include/net/dropreason.h
-// Here we only need SKB_DROP_REASON_NOT_SPECIFIED
-enum skb_drop_reason {
-	SKB_NOT_DROPPED_YET = 0,
-	SKB_CONSUMED,
-	SKB_DROP_REASON_NOT_SPECIFIED
-};
-
 SEC("tracepoint/skb/kfree_skb")
```

Comment:
I tested that it works fine on my kernel (`5.18.9-200.fc36.x86_64`):
```
$ sudo bpftool btf  dump id 1 format c| grep -A 50 'enum skb_drop_reason {'
enum skb_drop_reason {
        SKB_NOT_DROPPED_YET = 0,
        SKB_DROP_REASON_NOT_SPECIFIED = 1,
...
	SKB_DROP_REASON_QDISC_DROP = 34,
```
And indeed it matches [include/linux/skbuff.h#L316-L318](https://github.com/torvalds/linux/blob/v5.18/include/linux/skbuff.h#L316-L318):
```
enum skb_drop_reason {
        SKB_NOT_DROPPED_YET = 0,
        SKB_DROP_REASON_NOT_SPECIFIED,  /* drop reason is not specified */
...
	SKB_DROP_REASON_QDISC_DROP,	/* dropped by qdisc when packet
					 * outputting (failed to enqueue to
					 * current qdisc)
					 */
```

Then, tcpdrop shows `QDISC_DROP` from id 34 correctly.